lib/multi_blas_quda.cu

```diff
@@ -7,6 +7,8 @@ namespace quda {
 
   namespace blas {
 
+    using csfield_ref_vec = std::vector<ColorSpinorField_ref>;
```

Comment:
What's the reason for defining this here and not in the corresponding header?

Reply:
This short hand isn't used anywhere except in the multi-blas, so to increase local readability, I figured this was the best thing to do.